kernel/src/main/scala/cats/kernel/std/double.scala

```diff
@@ -0,0 +1,26 @@
+package cats.kernel
+package std
+
+import java.lang.Math
+
+trait DoubleInstances {
+  implicit val doubleOrder: Order[Double] = new DoubleOrder
+}
+
+class DoubleOrder extends Order[Double] {
```

Comment:
This implementation is using `java.lang.Double.compare` which means it is total (but ugly) -- `NaN` is greater than all non-NaN values:

```
scala> java.lang.Double.compare(Double.NaN, Double.PositiveInfinity)
res4: Int = 1
scala> java.lang.Double.compare(Double.NaN, Double.NaN)
res7: Int = 0
```